models/project/project.go

```diff
@@ -19,22 +19,22 @@ type Project interface {
 }
 
 type ProjectStruct struct {
-	Id               int64
-	UserId           int64
+	ID               int64
+	UserID           int64
 	Title            string
 	Description      string
-	RepositoryId     sql.NullInt64
+	RepositoryID     sql.NullInt64
 	ShowIssues       bool
 	ShowPullRequests bool
 	database         db.DB
 }
 
-func NewProject(id int64, userID int64, title string, description string, repositoryId sql.NullInt64, showIssues bool, showPullRequests bool) *ProjectStruct {
+func NewProject(id int64, userID int64, title string, description string, repositoryID sql.NullInt64, showIssues bool, showPullRequests bool) *ProjectStruct {
```

Comment:
__[golint]__
* [golint] exported function NewProject should have comment or be unexported